src/components/UI/Buttons/CircleCloseButton/CircleCloseButton.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+import Image from 'next/image'
+import CloseIcon from '@/../public/close.svg'
+
+export default function CircleCloseBtn({ onClick }: { onClick: () => void }) {
+  return (
+    <div onClick={onClick} onKeyDown={onClick}>
+      <button
```

Comment:
You added onClick and onKeyDown to the div instead of a button because of an accessibility concerns due to the size of this button, right? or there was another reason?

Reply:
I just make it similar to CircleAddBtn and didn't notice, I will fix it

Reply:
Fixed